gomock/call.go

```diff
@@ -288,7 +296,7 @@ func (c *Call) isPreReq(other *Call) bool {
 }
 
 // After declares that the call may only match after preReq has been exhausted.
-func (c *Call) After(preReq *Call) *Call {
+func (c *Call) After(preReq WrapperCall) WrapperCall {
```

Comment:
This change seems like I could break something. After returns a *Call that can be chained and used by something else. If a consumer of the result of After() used a method of *Call that is not covered by WrapperCall then something would break. We should at least add some test to cover something like this as well.